mslib/static/templates/user/reset_confirmation.html

```diff
@@ -0,0 +1,7 @@
+<p>Reset your password?</p>
```

Comment:
We should send this as text.

So that everyone can read what is behind the button.

e.g. 

Someone has requested to email you a password reset link for {{username}} 

If you have lost your password, please click the password reset URL .